packages/frontend/app/routes/assets.$assetId.add-liquidity.tsx

```diff
@@ -0,0 +1,79 @@
+import { type ActionArgs } from '@remix-run/node'
```

Comment:
is there a way where we can add folders for the routes? i.e. `routes/assets/*` and `routes/peers/*`?

Reply:
https://remix.run/docs/en/main/file-conventions/route-files-v2#folders-for-organization

Reply:
Yes, I'll reorganize them in a PR that will close https://github.com/interledger/rafiki/issues/1464.